spec/requests/providers/own_home_spec.rb

```diff
@@ -12,78 +12,102 @@
 
   describe 'PATCH providers/own_home' do
     let(:own_home) { 'owned_outright' }
-    let(:params) { { legal_aid_application: { own_home: own_home } } }
-    before { patch providers_legal_aid_application_own_home_path(legal_aid_application), params: params }
-
-    it 'updates the record' do
-      expect(legal_aid_application.reload.own_home).to eq own_home
+    let(:params) do
+      {
+        legal_aid_application: { own_home: own_home },
+        'continue-button' => 'Continue'
+      }
     end
 
-    context 'owned outright' do
-      xit 'redirects to 1b. How much is your client’s home worth' do
-        # TODO: write a spec when 1b build
-      end
+    context 'Continue button pressed' do
+      before { patch providers_legal_aid_application_own_home_path(legal_aid_application), params: params }
 
-      it 'displays the 1b holding page' do
-        expect(response).to have_http_status(:ok)
-        expect(response.body).to include('Holding page')
-        expect(response.body).to include('1b')
+      it 'updates the record' do
+        expect(legal_aid_application.reload.own_home).to eq own_home
       end
 
-      it 'updates the record to match' do
-        expect(legal_aid_application.reload.own_home_owned_outright?).to be_truthy
+      context 'owned outright' do
+        it 'redirects to the property value page' do
+          expect(response).to redirect_to providers_legal_aid_application_property_value_path(legal_aid_application)
+        end
+
+        it 'updates the record to match' do
+          expect(legal_aid_application.reload.own_home_owned_outright?).to be_truthy
+        end
       end
-    end
 
-    context 'mortgaged' do
-      let(:own_home) { 'mortgage' }
+      context 'mortgaged' do
+        let(:own_home) { 'mortgage' }
 
-      xit 'redirects to 1b. How much is your client’s home worth' do
-        # TODO: write a spec when 1b build
+        it 'redirects to the property value page' do
+          expect(response).to redirect_to providers_legal_aid_application_property_value_path(legal_aid_application)
+        end
+
+        it 'updates the record to match' do
+          expect(legal_aid_application.reload.own_home_mortgage?).to be_truthy
```

Comment:
This would also work:
```ruby
expect(legal_aid_application.reload).to be_own_home_mortgage
```

Reply:
Not my code - just showing up because now indented inside a new context.  Leaving it for now.

Reply:
done